spot_ros2_control/src/noarm_squat.cpp

```diff
@@ -0,0 +1,64 @@
+// based off: https://github.com/boston-dynamics/spot-sdk/blob/master/python/examples/joint_control/noarm_squat.py
+
+#include <chrono>
+#include <functional>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "rclcpp/rclcpp.hpp"
+#include "sensor_msgs/msg/joint_state.hpp"
+#include "std_msgs/msg/float64_multi_array.hpp"
+
+class NoarmSquat : public rclcpp::Node {
+ public:
+  NoarmSquat() : Node("noarm_squat"), standing{true}, initialized{false} {
+    declare_parameter("timer_rate", 5.0);
+    const auto timer_rate = std::chrono::duration<double>{get_parameter("timer_rate").as_double()};
+    command_pub_ = create_publisher<std_msgs::msg::Float64MultiArray>("/forward_position_controller/commands", 10);
+    joint_states_sub_ = create_subscription<sensor_msgs::msg::JointState>(
+        "joint_states", 10, std::bind(&NoarmSquat::joint_states_callback, this, std::placeholders::_1));
+    timer_ = create_wall_timer(timer_rate, std::bind(&NoarmSquat::timer_callback, this));
```

Comment:
Is rate the best name here?  The default here is once every 5 secs, but rate implies once every 5 seconds.  timer_interval_sec may be less ambigious.